### ISSUE TITLE
can fontdue rasterize with fractional pixel offsets?

### ISSUE THREAD

**Issue by wtholliday** [mooman219/fontdue#125]
I don't see anything in the API, but perhaps I'm missing it?

I don't need RGB subpixel AA (and indeed I don't think it would work well with my glyph atlas approach, especially when text is transformed), but it would be nice to rasterize with subpixel offsets.

I'm using fontdue in [this library](https://github.com/audulus/vger-rs) which is used in my [little GUI library](https://github.com/audulus/rui)

**Comment by mooman219** [comment]
It's not exposed. There are internals for it, but I haven't put much thought into designing the API. We're already up to 6 rasterize functions and I feel like it's getting out of hand. It honestly might make more sense to make a breaking version change and use a builder style for the various glyph parameters (and potentially future ones), but I'm not sure.

https://github.com/mooman219/fontdue/blob/master/src/font.rs#L574

That 3rd parameter set to 0.0 is actually the horizontal offset.

**Comment by wtholliday** [comment]
@mooman219 Builder sounds good. Let me know if I can help out!

**Comment by alokedesai** [comment]
Following up on this, was there any progress made on this? 

I'd be happy to help submit a PR for this if we can settle on an API that folks feel good about.

**Comment by alokedesai** [comment]
@mooman219 Ping on this, would you be open to accepting a PR to add support for this?

**Comment by mooman219** [comment]
Yeah feel free to play around with ideas and send a PR. If you want to explore what a builder might look like I'm open to that, otherwise we can take the easy way out with some more function bloat and expose the offset parameters in yet another function.
